frontend/src/App.js

```diff
@@ -58,6 +62,31 @@ const theme = createMuiTheme({
 });
 
 function App() {
+  const [isAuthenticated, userHasAuthenticated] = useState(false);
```

Comment:
There are certain rules of hooks that you should follow. When using `useState`, the second returned value should always start with `set` and and with however you named the first value. So in this case: `[isAuthenticated, setIsAuthenticated]`. This allows linters to properly check if hooks are used properly.